app/static/js/base.js

```diff
@@ -16,3 +16,44 @@ $(document).ready(function() {
         $("#userSelectForm").submit();
     });
 });
+
+function validatePhoneNumber(editButtonId, phoneInputId, username) {
+    var isvalid = null
+    if ($(editButtonId).html() === 'Edit') {
+        $(editButtonId).html("Save");
+        $(phoneInputId).focus();
+    } else {
+        // Save the phone number
+        var phoneInput = $(phoneInputId);
+        isvalid = phoneInput.val().replace(/\D/g,"").length === 10;
+        let isempty = phoneInput.val().replace(/\D/g,"").length === 0;
+        if (!(isvalid || isempty)) {
```

Comment:
Maybe we should have a comment here clarifying that this expression allows the phone input to be empty.